### PR TITLE
issue #34 Long links

### PR DESCRIPTION
**Pull request by mardub1635**
Enable time handling in youtube Long links.

### CODE REVIEW

File: desktop.js [hunk 1]
```diff
@@ -542,32 +541,48 @@ desktop.smartlinks = {};
 desktop.smartlinks.replaceYoutubeLinks = function (el) {
   let cleanText = el.html();
 
-  // still doesn't support https://www.youtube.com/watch?v=8JgxMVMBESY&t=33s style links ?
+  // TODO:
```
Comment (Marak) [inline desktop.js:544]: What is to TODO here?

**Comment by Marak** [comment]
Thank you for your contributions @mardub1635 

We've got the most common formats working and that should be OK for now.

The issue is that the code here for doing smartlink with time is growing too large for simple thing and has too many conditionals. If you can refactor the whole thing to use less lines of code we can merge it.

Will leave open for now.

File: desktop.js (hunk 1 above)
Comment (mardub1635) [reply desktop.js:544]: Factorise?

**Comment by Marak** [comment]
Better to just merge it now. 

I have refactor half way here, gave up after 10 minutes.

If it's working it's working. Thank you for your contributions.

**Comment by mardub1635** [comment]
No problem. Cheers, it is a cool project.

**Pull request merged**